ui/lib/kv/addon/components/page/list.hbs

```diff
@@ -0,0 +1,74 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @model={{@model}} @isEngineView={{true}} @pageTitle={{@model.backend}}>
+  <:tabLinks>
+    <LinkTo @route={{@routeName}} data-test-secrets-tab={{@routeName}}>Secrets</LinkTo>
+    <LinkTo @route="configuration" data-test-secrets-tab="Configuration">Configuration</LinkTo>
+  </:tabLinks>
+
+  <:toolbarFilters>
+    {{! TODO add navigateInput }}
+  </:toolbarFilters>
+
+  <:toolbarActions>
+    {{! TODO this create link route needs to handle creating from a directory e.g. beep/boop/  "initialKey"}}
+    <ToolbarLink @route="create" @type="add">Create secret</ToolbarLink>
+  </:toolbarActions>
+</KvPageHeader>
+{{! TODO add permissions on the menu items }}
+{{#if @model.arrayOfSecretModels}}
+  {{#each @model.arrayOfSecretModels as |secret|}}
+    <LinkedBlock
+      class="list-item-row"
+      @params={{array (if secret.pathIsDirectory "list-directory" "secret.details") secret.fullSecretPath}}
+      @linkPrefix={{this.mountPoint}}
+    >
+      <div class="level is-mobile">
+        <div class="level-left">
+          <div>
+            <Icon @name="user" class="has-text-grey-light" />
+            <span class="has-text-weight-semibold is-underline">
+              {{secret.path}}
+            </span>
+          </div>
+        </div>
+        <div class="level-right is-flex is-paddingless is-marginless">
+          <div class="level-item">
+            <PopupMenu>
+              <nav class="menu">
+                <ul class="menu-list">
+                  {{#if secret.pathIsDirectory}}
```

Comment:
I feel like this popup menu should just be removed for directories. 🤔 Clicking on the linked block to navigate seems sufficient, and feels more like navigating through a directory. The 'content' menu feels pretty redundant...I know we're trying not to make too many styling changes, so I understand if we want to keep it, just putting it out there! Probably more of a design question

Reply:
I agree. And went back and forth on this. I settled on keeping it at is because it's what current users expect, and if I remove any menu li's it changes the linkedblock row height and throws things off. Those could all be corrected, but that can be a nice-to-have after we get this sucker merge (and if we can even get this thing done in time).